app/models/course.rb

```diff
@@ -1,5 +1,7 @@
 class Course < ActiveRecord::Base
   acts_as_tenant(:instance)
+  # for automatic update userstamps
```

Comment:
I'd just remove this comment because practically every model would have this declaration.